README.md

```diff
@@ -59,6 +59,13 @@ let args = {
 vscode.commands.executeCommand("graphviz-interactive-preview.preview.beside", args)
 ```
 
+## Maintainers
```

Comment:
The description for the search API needs updates.

Reply:
added `search` field.